src/DI/ParsedownExtraExtension.php

```diff
@@ -6,47 +6,50 @@
 use Contributte\Parsedown\ParsedownFilter;
 use Nette\Bridges\ApplicationLatte\ILatteFactory;
 use Nette\DI\CompilerExtension;
+use Nette\DI\Definitions\FactoryDefinition;
 use Nette\DI\Statement;
 use Nette\InvalidStateException;
+use Nette\Schema\Expect;
+use Nette\Schema\Schema;
 
 /**
  * ParsedownExtra Extension
  */
 class ParsedownExtraExtension extends CompilerExtension
 {
 
-	/** @var mixed[] */
-	private $defaults = [
-		'helper' => 'parsedown',
-	];
+	public static function createSchema(): Schema
```

Comment:
Why do you need static fnc?

Reply:
It's copied from a package where we compose extensions into single root extension, not needed here